src/content/translations/pt-br/roadmap/merge/index.md

```diff
@@ -37,7 +37,7 @@ A integração representou a mudança oficial para o uso da Beacon Chain como o
 Nenhuma história foi perdida na Fusão. À medida que a Rede principal se uniu com a Beacon Chain, ela também integrou todo o histórico transacional do Ethereum.
 
 <InfoBanner>
-Essa transição para a prova de participação mudou o modo como o ether é emitido. Saiba mais sobre <a href="/roadmap/merge/issuance/">Emissão de ether antes de depois da Fusão</a>.
+Essa transição para a prova de participação mudou o modo como o ether é emitido. Saiba mais sobre <a href="/roadmap/merge/issuance/">Emissão de ether antes de depois do The Merge</a>.
```

Comment:
Not sure why they're using the english term here ('The Merge' instead of the previous translation)

Reply:
Not sure either, but this was coming in from crowdin so this should be resolved there.